getting-started/windows-calico/quickstart.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Quickstart
+description: Install Calico for Windows on a Kubernetes cluster for testing or development.
+canonical_url: '/getting-started/windows-calico/quickstart'
+---
+
+### Big picture
+
+Install {{site.prodnameWindows}} on your Kubernetes cluster in approximately 5 minutes.
+
+### Concepts
+
+{{site.prodnameWindows}} is a hybrid implementation that requires a Linux control node for {{site.prodname}} components, and a Windows cluster for Windows nodes.
+
+### Before you begin
+
+**Datastore requirements**
+Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Windows node/Kubernetes cluster must be the same as the datastore for the Linux control node. (You cannot mix datastores in a {{site.prodnameWindows}} implementation.)
+
+**Kubernetes cluster requirements**
+- K2 clusters with versions 1.17, 1.16, or 1.15
```

Comment:
@bmckercher123 is this a new term we're using for Kubernetes clusters?

Reply:
For OS, we now have K3 cluster documentation in /Install. As far as I know, we do not support K3. @song?

Reply:
I've changed this back to the original term